samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
FYI: Certificate.cer and Certificate.keys are only needed on CoreCLR (because .pfx are not supported) and Mono (because there's a bug in the `System.IdentityModel.Tokens` package).
I dunno if it's helpful to merge them if you don't plan to support these targets.